build.gradle

```diff
@@ -37,15 +37,15 @@ run {
 
 defaultTasks "build"
 
-sourceCompatibility = 1.7
-targetCompatibility = 1.7
+sourceCompatibility = 1.8
```

Comment:
It dosen`t matter. ... Anyone while the development I had compilation or runtime error. I checked this again with source- and target-level=1.7. It works.